Mage.Sets/src/mage/cards/a/AmuletOfVigor.java

```diff
@@ -76,6 +76,7 @@ public boolean checkTrigger(GameEvent event, Game game) {
     @Override
     public String getRule() {
         // that triggers depends on stack order, so make each trigger unique with extra info
+        // TODO: Do the hints belong here? Must be a better place for them.
```

Comment:
Maybe take similar approach to https://github.com/magefree/mage/commit/07902f64b77da784fb23b136324314e7540c7f99?